Bottom: 6cfe64536ee67ed956a1fb7a68cb32cdf3eb8221
Top:    1f5b25ecd6a90e4be81ba9dfb4ad907df20e0b6c
Author: Helge Deller <deller@gmx.de>
Date:   2013-03-08 13:44:05 +0800

autofs-5.0.7 - fix automounter support on parisc

This patch fixes automounter support on the parisc architecture with
64-bit kernel and 32-bit userspace.

Signed-off-by: Helge Deller <deller@gmx.de>


---

diff --git a/daemon/automount.c b/daemon/automount.c
index 4a3eb3d..4c651cf 100644
--- a/daemon/automount.c
+++ b/daemon/automount.c
@@ -610,6 +610,7 @@ static size_t get_kpkt_len(void)
 		if (strcmp(un.machine, "alpha") == 0 ||
 		    strcmp(un.machine, "ia64") == 0 ||
 		    strcmp(un.machine, "x86_64") == 0 ||
+		    strcmp(un.machine, "parisc64") == 0 ||
 		    strcmp(un.machine, "ppc64") == 0)
 			pkt_len += 4;
